src/cryptonote_protocol/quorumnet.cpp

```diff
@@ -564,6 +565,12 @@ void handle_obligation_vote(Message& m, QnetState& qnet) {
     }
 }
 
+void handle_timestamp(Message& m) {
+    MDEBUG("Received a timestamp request from " << to_hex(m.conn.pubkey()));
+    time_t seconds = time(NULL);
```

Comment:
`const time_t seconds = ...` (const correctness)

Reply:
also style nitch: use `nullptr` not `NULL`

Reply:
> also style nitch: use `nullptr` not `NULL`

The main reason for this one is that NULL is a macro usually defined to 0, which means you can pass `NULL` in places that will accept non-pointer values like integers.  `nullptr` is its own type (of `nullptr_t`) that is *only* implicitly convertible to pointer types.